site/src/pages/TemplateVersionEditorPage/ProvisionerTagsPopover.tsx

```diff
@@ -111,18 +111,13 @@ export const ProvisionerTagsPopover: FC<ProvisionerTagsPopoverProps> = ({
                     return key !== "owner";
                   })
                   .map((k) => (
-                    <>
+                    <Fragment key={k}>
                       {k === "scope" ? (
-                        <ProvisionerTag key={k} k={k} v={tags[k]} />
+                        <ProvisionerTag k={k} v={tags[k]} />
                       ) : (
-                        <ProvisionerTag
-                          key={k}
-                          k={k}
-                          v={tags[k]}
-                          onDelete={onDelete}
-                        />
+                        <ProvisionerTag k={k} v={tags[k]} onDelete={onDelete} />
                       )}
-                    </>
+                    </Fragment>
```

Comment:
this doesn't even need to be a fragment, both branches only return one element. I guess it's nice to only specify the key in one place tho.

Reply:
Yes!